index.js

```diff
@@ -1701,16 +1747,15 @@
     }
 
     function test(x) {
-      return x != null &&
```

Comment:
Indeed. It's similar to how we expect the `length` property to be present in types extending `String`.